src/main/java/emissary/core/channels/FileChannelFactory.java

```diff
@@ -28,14 +22,14 @@ private FileChannelFactory() {}
      * @return a new instance
      * @see SeekableByteChannelHelper#file(Path)
      */
-    public static SeekableByteChannelFactory create(final Path path) {
-        return ImmutableChannelFactory.create(new FileChannelFactoryImpl(path));
+    public static ImmutableChannelFactory<?> createFactory(final Path path) {
```

Comment:
This method rename is one I repeated across the other `XXXChannelFactory` classes as well, for the selfish reason that I makes it easier for me to read some chained calls:
 - instance methods that return a channel instance are still named `create`
 - static methods that return a channel _factory_ instance are now named `createFactory`, rather than `create`